docs/pages/versions/unversioned/sdk/ui.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: UI
+description: A set of platform components to build native-feeling UIs.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-ui'
+packageName: '@expo/ui'
+platforms: ['android', 'ios']
+---
+
+import APISection from '~/components/plugins/APISection';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+
+`@expo/ui` is a set of native input components that allows you to build native-feeling forms.
+
+> **warning** This library is currently in alpha and subject to breaking changes. It is not available in the Expo Go app.
+
+## Installation
+
+<APIInstallSection />
+
+## Examples
+
+### Button
+
+<Tabs>
+<Tab label="iOS Screenshot">
```

Comment:
```suggestion
<Tab label="iOS">
```

Let's also remove screenshot from every tab label. It's self-explanatory that it's a screenshot and we need only need to mention the platform name.